app/adapters/alma/adapter.rb

```diff
@@ -0,0 +1,36 @@
+module Alma
+  class Adapter
```

Comment:
Should this class be called `Connector`? The `Bib` class seems more likely to be the place to provide all the business logic methods, and it's the one used as `bib_adapter` in the controller so it seems confusing to call this class `adapter`.